ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/daemon-terminattr.j2

```diff
@@ -1,75 +1,98 @@
 {# eos - TerminAttr #}
 {% if daemon_terminattr is arista.avd.defined %}
-!
-{# Create list of non CVaaS server #}
-{%     set cvp_on_prem = namespace() %}
-{%     set cvp_on_prem.ips = [] %}
-{%     set cvaas = namespace() %}
-{%     set cvaas.ips = [] %}
 {%     set cvp_config = namespace() %}
-{%     for cvp_ip in daemon_terminattr.ingestgrpcurl.ips %}
-{%         if "arista.io" not in cvp_ip %}
-{%             do cvp_on_prem.ips.append(cvp_ip) %}
-{%         elif "arista.io" in cvp_ip %}
-{%             do cvaas.ips.append(cvp_ip) %}
-{%         endif %}
-{%     endfor %}
+!
 daemon TerminAttr
-{%     if cvp_on_prem.ips | length > 0 %}
-{%         set cvp_config.cli = "exec /usr/bin/TerminAttr -ingestgrpcurl=" %}
-{%         for cvp_ip in cvp_on_prem.ips %}
-{%             set cvp_config.cli = cvp_config.cli ~ cvp_ip ~ ":" ~ daemon_terminattr.ingestgrpcurl.port %}
-{%             if not loop.last %}
-{%                 set cvp_config.cli = cvp_config.cli ~ "," %}
-{%             endif %}
+{%     set cvp_config.cli = "exec /usr/bin/TerminAttr" %}
+{%     if daemon_terminattr.clusters | arista.avd.natural_sort %}
```

Comment:
this `if` can be removed. The `arista.avd.natural_sort` used in the `for` loop below has this test builtin. Remember to adjust spacing.

Reply:
ack